README.md

```diff
@@ -166,8 +166,21 @@ plan_hierarchy:
     data_hash: yaml_data
 ```
 
+
 See the following [documentation](https://puppet.com/docs/bolt/latest/hiera.html#outside-apply-blocks) for further explanation.
 
+#### Using indictor_exclusion with a code manager and hiera.yaml
+
+To utilize the indicator_exclusions with a code manager, place the plan_hierarchy listed in the step above, placing it in the root level [hiera.yaml](https://puppet.com/docs/puppet/latest/hiera_config_yaml_5.html) file.
+
+Following this create a [static.yaml] file within the code manager data folder `CODE_MANAGER/data/static.yaml`
+```yaml
+pe_status_check::indicator_exclusions:                                             
+  - '<TEST ID>'                                                                
+``` 
+
+This will allow you to exclude tests from `pe_status_check::infra_summary` and `pe_status_check::agent_summary` within the PE Console planning
```

Comment:
This sentence needs some work,

indicator ids in this array will be excluded when running the plans  `pe_status_check::infra_summary` and `pe_status_check::agent_summary`

something like that